packages/terra-form-select/src/combobox/Frame.jsx

```diff
@@ -396,11 +402,13 @@ class Frame extends React.Component {
       'aria-describedby': ariaDescribedBy,
       'aria-disabled': disabled,
       'aria-owns': this.state.isOpen ? id : undefined,
+      'aria-expanded': !disabled && this.state.isOpen,
       type: 'text',
       className: cx('search-input'),
       required,
       'aria-required': required,
       id: inputId,
+      role: this.role(),
```

Comment:
May I know the reason for adding role `combobox` for input ..?

Reply:
Hi @supreethmr . 
Now combo-box component reads as application which we need to change to combobox role. And as per W3C example we updated this input field role to combo box.
W3C Example -- https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-autocomplete-both/
Thanks